modules/installation-configuring-nutanix-failure-domains.adoc

```diff
@@ -45,7 +45,7 @@ where:
 `<failure_domain_name>`:: Specifies a unique name for the failure domain. The name is limited to 64 or fewer characters, which can include lower-case letters, digits, and a dash (`-`). The dash cannot be in the leading or ending position of the name.
 `<prism_element_name>`:: Optional. Specifies the name of the Prism Element.
 `<prism_element_uuid`>:: Specifies the UUID of the Prism Element.
-`<network_uuid`>:: Specifies the UUID of the Prism Element subnet object. The subnet's IP address prefix (CIDR) should contain the virtual IP addresses that the {product-title} cluster uses. Only one subnet per failure domain (Prism Element) in an {product-title} cluster is supported.
+`<network_uuid`>:: Specifies the UUID of the Prism Element subnet object. The subnet's IP address prefix (CIDR) should contain the virtual IP addresses that the {product-title} cluster uses. A maximum of 32 subnets for each failure domain (Prism Element) in an {product-title}  cluster is supported. All `subnetUUID` values must be unique.
```

Comment:
Better change to something like below:

Specifies the one or more UUIDs of the Prism Element subnet objects. Among them, one of the subnet's IP address prefix (CIDR) should contain the virtual IP addresses that the {product-title} cluster uses. A maximum of 32 subnets for each failure domain (Prism Element) in an {product-title}  cluster is supported. All `subnetUUID` values must be unique.